scripts/build/get-version.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
```

Comment:
Please add `set -e` here, so that script will exit if any command fails.

Reply:
Done. PTAL